### PR TITLE
Unary operators

### PR DESCRIPTION
**Pull request by manoelrui**
@gunthercox I am trying to fix the bug #13, I added more details in there.

### CODE REVIEW

File: mathparse/mathparse.py [hunk 1]
```diff
@@ -96,7 +96,18 @@ def replace_word_tokens(string, language):
 
     # Replace operator words with numeric operators
     operators = words['binary_operators'].copy()
-    operators.update(words['unary_operators'])
+    if u'unary_operators' in words:
+        operators.update(words['unary_operators'])
+
+    # Ensures unicode string processing
```
Comment (gunthercox) [inline mathparse/mathparse.py]: I'm not sure if decoding the string is necessary. Was there a case where this was failing?

File: mathparse/mathparse.py [hunk 2]
```diff
@@ -96,7 +96,18 @@ def replace_word_tokens(string, language):
...
+    import sys
+    if sys.version_info.major == 3:
+        if not isinstance(string, str):
+            string.decode("utf-8")
```
Comment (gunthercox) [inline mathparse/mathparse.py]: This `decode` line might not be doing anything. I believe it [returns the decoded value](https://docs.python.org/3/library/stdtypes.html#bytes.decode) and this line doesn't assign it with `string =` like the one bellow.

Comment (manoelrui) [reply mathparse/mathparse.py]: @gunthercox Good observation, I think this code is not necessary too because in python3 strings already are unicode.

**Comment by manoelrui** [comment]
@gunthercox Thank you for observations, do you have any additional suggestion to improve the code ?

**Comment by gunthercox** [comment]
This looks good to me 👍 I'll merge it in.

Thank you for taking the time to work on this.

**Comment by manoelrui** [comment]
Thank you too @gunthercox, I learned a lot during the process and I am glad to help.

**Pull request merged**